rasa/shared/core/events.py

```diff
@@ -345,6 +345,18 @@ def __eq__(self, other: Any) -> bool:
         return True
 
 
+class SkipEventInMDStoryMixin(Event, ABC):
```

Comment:
do we need that? let's just add `as_story_string` to all the methods

Reply:
@joejuzl what do you think. this method seems to me redundant but also increases cognitive complexity of events

Reply:
I think it's fine. It means you can easily identify events which won't be in the story rather than having to look at the implementation of the method in each event.

Reply:
to me it contradicts `Occam’s razor` principle, but after all it's your part of code

Reply:
for me personally, it makes it harder to understand, I see the method - I know how it is represented, looking at superclass requires more cognitive function

Reply:
I see arguments for both ways - but in the interest of speed (and since it's already done like this) - I think we should keep it as is.